example/uwsgi_setup/uwsgi/uwsgi.ini.docker

```diff
@@ -14,7 +14,7 @@ processes   = 8
 # sets max connections to
 listen = 2048
 
-wsgi-file   = /usr/lib/python3.11/site-packages/satosa/wsgi.py
+wsgi-file   = /.venv/lib/python3.11/site-packages/satosa/wsgi.py
```

Comment:
@salvatorelaiso could you please check if is there any way to ref python version withing the uwsgi ini file?

I would like to avoid this constant about the python version used runtime

Reply:
@peppelinux what do you think about this: https://github.com/italia/Satosa-Saml2Spid/pull/117/commits/f454232bf6a4597b657f21fc88587ee4195f2e20 ?